### DISCUSSION TITLE
CMCC RAX3000M 固件的选择

### DISCUSSION THREAD

**Discussion by CrazyBoyFeng** [immortalwrt/immortalwrt#1110] [Q&A]
我的机器是 128M 的 NAND 版。

在 Firmware Selector 中，选择版本 `23.05.0-rc4`，输入 CMCC RAX3000M，可供选择的固件有：
1. CMCC RAX3000M (OpenWrt U-Boot layout)
2. CMCC RAX3000M eMMC version (custom U-Boot layout)
3. CMCC RAX3000M NAND version (custom U-Boot layout)

第二个和第三个声明为定制 uboot 布局，但是并没有提供相关 uboot 文件和布局文件。
请问什么时候选择第一个，什么时候选择第三个或第二个？

**Comment by fyang93** [comment]
同样有这个疑问

**Reply by CrazyBoyFeng** [reply]
通过与别人的交流，我得知 custom U-Boot layout 的固件，仅适用于 hanwckf 制作的 uboot fip。

**Comment by 1715173329** [comment]
不知道就用 OpenWrt U-Boot layout，其他两个为废弃方案。

**Reply by bywayboy** [reply]
算力版 刷完系统后  丢失了57G空间。 有什么办法能将rootfs 扩容下么？ 求教。
